src/main/java/ipman/Deadline.java

```diff
@@ -0,0 +1,35 @@
+package ipman;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Deadline class
+ * @author miloaisdino
+ */
+public class Deadline extends Task {
+    protected LocalDateTime by;
+
+    /**
+     * Constructor
+     * @param description Name of deadline
+     * @param stringDate String in DD/MM/YYYY hh[0-23]mm format
+     * @throws CommandException Error
```

Comment:
This comment applies for all your Javadoc comments that violate this coding standard. There should be an empty line between the description and parameter section.